README.adoc

```diff
@@ -146,233 +285,40 @@ If you would like to modify the container names, you can use the following envir
 
 |===
 
-
 === Backups
 
 The `BACKUP_IMAGE` environment variable configures what image to use for backing up
 the custom resources created by this operator. Default value is `quay.io/integreatly/backup-container:1.0.8`.
 
-
-=== Metrics
+=== Monitoring Service (Metrics)
 
 The application-monitoring stack provisioned by the
-{application_monitoring_operator} can be used to gather metrics from
-the operator here.  Once you have provisioned that (or the
-ServiceMonitor CRD at a minimum), you can run the following commands
-to configure it:
+{application_monitoring_operator} on https://github.com/integr8ly[Integr8ly]
+can be used to gather metrics from this operator and the UnifiedPush Server. These metrics can be used by Integr8ly's application monitoring to generate Prometheus metrics, AlertManager alerts and a Grafana dashboard.
 
-....
-kubectl label namespace unifiedpush monitoring-key=middleware
-kubectl create -n unifiedpush -f deploy/service_monitor.yaml
-....
+It is required that the https://github.com/integr8ly/grafana-operator[integr8ly/Grafana] and https://github.com/coreos/prometheus-operator[Prometheus] operators are installed. For further detail see https://github.com/integr8ly/application-monitoring-operator[integr8ly/application-monitoring-operator].
 
-== Custom Resources (aka How to get value from this operator)
+The following command enables the monitoring service in the operator namespace:
 
-=== UnifiedPushServer
```

Comment:
@camilamacedo86 I think the doc for this custom resource is missing.

Reply:
OK.

Reply:
done